workflow-runtime/src/main/java/com/squareup/workflow1/internal/WorkflowRunner.kt

```diff
@@ -57,9 +58,11 @@ internal class WorkflowRunner<PropsT, OutputT, RenderingT>(
    * between every subsequent call to [nextOutput].
    */
   fun nextRendering(): RenderingAndSnapshot<RenderingT> {
-    val rendering = rootNode.render(workflow, currentProps)
-    val snapshot = rootNode.snapshot(workflow)
-    return RenderingAndSnapshot(rendering, snapshot)
+    return Tracing.trace({ "Render Pass" }) {
+      val rendering = rootNode.render(workflow, currentProps)
+      val snapshot = rootNode.snapshot(workflow)
+      RenderingAndSnapshot(rendering, snapshot)
+    }
   }
```

Comment:
Here is where we trace the 'render pass'. Still open to feedback about other places we might want to have pluggable tracing.

I'm also thinking about possibly considering a 'cumulative' render time in the benchmark some way. So if we keep the scenarios the same we can see improvements from lessening churn.